### PR TITLE
Sort Imports: pay attn to all pragmas

### PR DESCRIPTION
**Pull request by samouri**
Intended to fix the issue where pragma docblocks and `External Dependencies` comments get merged.

This PR upgrades the `sort-imports` codemod so that it makes sure the Deps block and the pragma block are separate.  It also now keeps all the old pragmas (jest) instead of just the format one.

e.g.: https://github.com/Automattic/wp-calypso/pull/18453#discussion_r142367916

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "inherits": "2.0.1",
     "is-my-json-valid": "2.13.1",
     "jade": "pugjs/jade#29784fd",
+    "jest-docblock": "21.3.0-alpha.eff7a1cf",
```
Comment (gziolo) [inline package.json]: Shouldn’t it land in dev deps?

Comment (samouri) [reply package.json]: sure. i'll move

**Comment by samouri** [comment]
> What about shrinkwrap? Should it get updated?
good point @gziolo, i'll update it.  can't wait for npm to start handling this on its own

**Comment by samouri** [comment]
Thanks for the thorough review @jsnmoon!!

> eslint-disable comments at the top of file gets stripped out.
> Any other top comment blocks get wiped out.

I noticed that issue with the codemod as well.  Its definitely fixable.  I'm not too worried because we can run prettier first which will prepend a docblock.  Then there would only be an issue if the top comment was formatted as a docblock.

**Pull request merged**